app/models/property.rb

```diff
@@ -75,6 +76,19 @@ class Property < ApplicationRecord
   tag_columns :prohibited_advertisers
   tag_columns :keywords
   has_one_attached :screenshot
+  has_paper_trail on: %i[create update destroy], only: %i[
+    user_id
+    property_type
+    status
+    name
+    url
+    ad_template
+    ad_theme
+    language
+    keywords
+    prohibited_advertisers
+    prohibit_fallback_campaigns
+  ]
```

Comment:
I'm might consider sorting these types of lists. Sorting makes the semantics worse, but it also helps speed up scanning for a value and spotting dupes in the list.